### PR TITLE
Django tutorial + Nav update

### PR DESCRIPTION
**Pull request by alvinometric**
Updating the nav so Django and Rails are both listed under their respective language.

### CODE REVIEW

File: website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+pip install UnleashClient
+```
+
+Then, create `blog/unleash_client.py`, and add the following:
+
+```python
+from UnleashClient import UnleashClient
+
+unleash_client = UnleashClient(
+    url="http://localhost:4242/api/",
```
Comment (sighphyre) [inline website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md:146]: I think it's worth just setting the  `refresh_interval` down to something short like 1 second and scrapping the warning about 30 seconds at the end. This is a tutorial, I think the fast updates are nice so long as the user is warned it's not a production ready config.

File: website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+First, install the `UnleashClient` package:
+
+```sh
+pip install UnleashClient
+```
+
+Then, create `blog/unleash_client.py`, and add the following:
+
+```python
+from UnleashClient import UnleashClient
```
Comment (sighphyre) [inline website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md:143]: Starting the client in the view is definitely not sane for a production config. I think that's totally sane for a tutorial and going through a dance of setting up a singleton is probably out of scope here but we should make this clear that this does belong in a singleton and not newed up where ever

**Review by sighphyre — CHANGES_REQUESTED** [review]
Nice! Looks like a good start

File: website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+![Create a new feature flag](../ruby/new-ff.png)
+
+Call it `top-3` and enable it in the `development` environment.
+
+![A feature flag called `top-3` is now visible.](../rails/enable-ff.png)
+
+Everything's now set up on the Unleash side. Let's set up the Django application.
+
+## 3. Setup the Django app
```
Comment (melindafekete) [inline website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md:76]:
```suggestion
## 3. Set up the Django app
```

File: website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+
+Let's clone a basic blog repository and get it up and running. We don't want to waste time setting up a Django codebase from scratch.
+
+```sh
+git clone https://github.com/alvinometric/django-basic-blog
+cd django-basic-blog
+```
+
+#### Setup/seed the database
```
Comment (melindafekete) [inline website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md:85]:
```suggestion
#### Set up and seed the database
```

File: website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+---
+
+Hello! In this tutorial, we’ll show you how to add feature flags to your Django app, using [Unleash](https://www.getunleash.io/) and the official [Unleash Python SDK](https://docs.getunleash.io/reference/sdks/python). With Unleash, an open-source feature flag service, you can use our tooling to add feature flags to your application and release new features faster.
+
+In a classic tutorial fashion, we’ll add feature flags to a blog app made with Django. We’ll use feature flags to decide how many blog posts to show on the index page.
+
+-   [Prerequisites](#prerequisites)
+-   [1. Best practices for back-end apps with Unleash](#1-best-practices-for-back-end-apps-with-unleash)
+-   [2. Install a local feature flag provider](#2-install-a-local-feature-flag-provider)
+-   [3. Setup the Django app](#3-setup-the-django-app)
```
Comment (melindafekete) [inline website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md:14]:
```suggestion
-   [3. Set up the Django app](#3-set-up-the-django-app)
```

File: website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+    -   [Setup/seed the database](#setupseed-the-database)
```
Comment (melindafekete) [inline website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md:15]:
```suggestion
    -   [Set up and seed the database](#set-up-and-seed-the-database)
```

**Review by melindafekete — COMMENTED** [review]
Thanks @alvinometric 🚀

File: website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+Log in to the platform using these credentials:
+
+```
+Username: admin
+Password: unleash4all
+```
+
+Click the 'New feature flag' button to create a new feature flag.
+
+![Create a new feature flag](../ruby/new-ff.png)
```
[the remark this answers is not in the document]
Comment (alvinometric) [reply website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md:68]: I agree, but we do this everywhere. I can make another PR to address that

[the remark this answers is not in the document]
Comment (sighphyre) [reply website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md:46]: Sure, can't complain if it's an existing pattern

**Review by sighphyre — APPROVED** [review]
:+1:

File: website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+
+Then, in `blog/views.py`, update the `post_list` view:
+
+```python
+from django.shortcuts import render
+from .models import Post
+from .unleash_client import unleash_client
+
+def post_list(request):
+    if  unleash_client.is_enabled("top-3"):
```
Comment (melindafekete) [inline website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md:149]:
```suggestion
    if unleash_client.is_enabled("top-3"):
```

File: website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+
+Modify the `post_list` view in `blog/views.py` to look like this:
+
+```python
+from django.shortcuts import render
+from .models import Post
+
+def post_list(request):
+    is_top3 = True
+    posts = Post.objects.order_by('-published_at')[:3] if is_top3 else Post.objects.all()
```
Comment (melindafekete) [inline website/docs/feature-flag-tutorials/django/implementing-feature-flags-django.md:103]:
```suggestion
    posts = Post.objects.order_by('-published_date')[:3] if is_top3 else Post.objects.all()
```

**Review by melindafekete — APPROVED** [review]

**Pull request merged**